src/Forms/PersonalDetails.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core/Typography";
+import Radio from "@material-ui/core/Radio";
+import RadioGroup from "@material-ui/core/RadioGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import FormControl from "@material-ui/core/FormControl";
+import FormLabel from "@material-ui/core/FormLabel";
+
+const PersonalDetails = ({ visible, firstName, lastName, email, gender, handleChange, handleBlur }) => 
+  visible ? 
+    <form className="form-spacer">
+      <Typography variant="h6" gutterBottom>
+        Personal Details
+      </Typography>
+      <div className="form-field">
+        <TextField 
+          name="firstName"
+          label="First Name *"
```

Comment:
What's the asterisk mean?

Reply:
required

Reply:
![image](https://user-images.githubusercontent.com/16544821/56787975-fabc6100-67b2-11e9-98d6-9f313c4f7076.png)

Reply:
Is that bad UX? nonsensical ux lol?